dlotter/arguments.py

```diff
@@ -95,11 +95,16 @@ def get_args(self, sysargs):
                         help='Verbose output', 
                         default=False)
 
+        parser_plot.add_argument('--dataset',
+                        type=str,
+                        help='Set to DANRAmm if DANRA monthly means are used.',
+                        default='',
+                        required=False)
```

Comment:
Either use `default=None` (empty strings are a bit dangerous) or set the variable as `store_true` so it can be used as a boolean flag instead.